cli/internal/hashing/package_deps_hash.go

```diff
@@ -86,6 +86,22 @@ func GetPackageDeps(rootPath fs.AbsolutePath, p *PackageDepsOptions) (map[turbop
 	return result, nil
 }
 
+func manuallyHashFiles(rootPath turbopath.AbsoluteSystemPath, files []turbopath.AnchoredSystemPath) (map[turbopath.AnchoredUnixPath]string, error) {
+	hashObject := make(map[turbopath.AnchoredUnixPath]string)
+	for _, file := range files {
+		hash, err := fs.GitLikeHashFile(file.ToString())
+		if err != nil {
+			return nil, fmt.Errorf("could not hash file %v. \n%w", file.ToString(), err)
+		}
+
+		if err != nil {
```

Comment:
This line looks wrong. Copied from elsewhere?

Reply:
Ah yep - we do a similar process for task hashes but we didn't need to do an extra file path conversion here that I removed.